strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/users/UserController.java

```diff
@@ -39,56 +44,66 @@
     @ApiResponses(value = { @ApiResponse(code = 200, message = ""),
                             @ApiResponse(code = 400, message = "An error occurred.") })
```

Comment:
The message here should probably be `Bad request` instead of the vague `An error occurred`. I'm not sure in which case this would return `Status: 400` to begin with. If it's never returning `400` we should just remove it.